File: backend/node-server/scratch/check_db.js
const sqlite3 = require('better-sqlite3');
const path = require('path');

const dbPath = 'd:\\JC\\Personal\\projects\\TvApp-JJtv\\JJTVAPI\\backend\\jjtv_config.db';
const db = new sqlite3(dbPath);

console.log('--- Media Cache Entries ---');
const rows = db.prepare('SELECT vpath, playlist, filename FROM media_cache LIMIT 10').all();
rows.forEach(row => {
    console.log(`Playlist: ${row.playlist} | Filename: ${row.filename} | Path: ${row.vpath}`);
});

console.log('\n--- Settings ---');
const settings = db.prepare('SELECT * FROM settings').all();
settings.forEach(s => {
    console.log(`${s.key}: ${s.value}`);
});

File: backend/node-server/scratch/check_db.py
import sqlite3
import os

db_path = r'd:\JC\Personal\projects\TvApp-JJtv\JJTVAPI\backend\jjtv_config.db'
if not os.path.exists(db_path):
    print(f"DB not found at {db_path}")
    exit(1)

conn = sqlite3.connect(db_path)
cursor = conn.cursor()

print('--- Media Cache Entries ---')
try:
    cursor.execute('SELECT vpath, playlist, filename FROM media_cache LIMIT 10')
    rows = cursor.fetchall()
    for row in rows:
        print(f"Playlist: {row[1]} | Filename: {row[2]} | Path: {row[0]}")
except Exception as e:
    print(f"Error reading media_cache: {e}")

print('\n--- Settings ---')
try:
    cursor.execute('SELECT * FROM settings')
    rows = cursor.fetchall()
    for row in rows:
        print(f"{row[0]}: {row[1]}")
except Exception as e:
    print(f"Error reading settings: {e}")

conn.close()
